crates/wysiwyg/src/composer_model/quotes.rs

```diff
@@ -311,13 +311,6 @@ mod test {
         );
     }
 
-    #[test]
-    fn remove_quote_containing_code_block() {
-        let mut model = cm("<blockquote>~<pre>~Some| code</pre></blockquote>");
-        model.quote();
-        assert_eq!(tx(&model), "<pre>~Some| code</pre>");
-    }
-
```

Comment:
Removed this as I didn't think this would be allowed from our conversation today @jmartinesp . Let me know if I'm wrong here.

Reply:
A quote can contain a code block, but not the other way around:

```
<blockquote><p>This will become code|</p></blockquote>
// Becomes
<blockquote><pre>This will become code|</pre></blockquote>
```

Reply:
Oh, never mind 🤦 . Given this change, it's impossible to do this without removing the code block first or creating a new paragraph outside of it. I think that there should be a way to do this, but we haven't really explored these cases so this is fine.